src/palace/manager/api/millenium_patron.py

```diff
@@ -187,8 +153,6 @@ def library_settings_class(cls) -> type[MilleniumPatronLibrarySettings]:
     RECORD_NUMBER_FIELD = "p81"  # e.g., "RECORD #[p81]"
     PATRON_TYPE_FIELD = "p47"  # e.g., "P TYPE[p47]"
     EXPIRATION_FIELD = "p43"  # e.g., "EXP DATE[p43]"
-    HOME_BRANCH_FIELD = "p53"  # e.g., "HOME LIBR[p53]"
```

Comment:
I was a little worried about `HOME_BRANCH_FIELD` here, since the "p53" can be used in patron auth restrictions, but it doesn't use this constant and eliminating this constant won't affect how the ILS properties are modeled. So, this looks fine.